lib/task/migrate/arUpgradeSqlTask.class.php

```diff
@@ -507,7 +507,7 @@ private function getNewTheme($themes)
             throw new Exception('This task needs the PHP readline extension.');
         }
 
-        for (;;) {
+          while (true){
```

Comment:
Indentation? :thinking:

Reply:
Weird! Check the CI error in qa/2.x @jraddaoui. Should it be using PHP 8.1?